src/styles/blocks/product.scss

```diff
@@ -0,0 +1,70 @@
+.product {
+  display: flex;
+  flex-direction: column;
+  color: $main-font-color;
+
+  &__image {
+    background-image: url(/src/images/imac.jpeg);
+    height: 134px;
+    width: 160px;
+    background-size: 100% 100%;
+    margin-bottom: 40px;
+    margin-inline: auto;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    margin-block: 0 4px;
+  }
```

Comment:
According to the code style rules, `font-size` should be defined before `font-weight`. Please rearrange the properties to follow the order.